content/swiftui/concepts/views/terms/circle/circle.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'Circle'
+Description: 'Pre-built shape which can be colored and positioned as desired.'
```

Comment:
```suggestion
Description: 'Pre-built circular shape which can be colored and positioned as desired.'
```

Reply:
Instead if saying "colored and positioned" can we say "visually modified?" This makes it broad as to what we can change about the circle view.